sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -1603,7 +1617,7 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
     val fs = path.getFileSystem(sparkContext.hadoopConfiguration)
     // The partition dir is not empty.
     assert(fs.listStatus(new Path(path, "part=1")).nonEmpty)
-
+    sql("DROP TABLE test_table")
```

Comment:
We don't need to do this. We will `DROP DATABASE test_database CASCADE` at [here](https://github.com/apache/spark/pull/19155/files#diff-1ea02a6fab84e938582f7f87cc4d9ea1R1622).